pulumi/grapl/__main__.py

```diff
@@ -189,27 +189,25 @@ def main() -> None:
     engagement_creator_queue.subscribe_to_emitter(analyzer_matched_emitter)
 
     analyzers_bucket = Bucket("analyzers-bucket", sse=True)
-    pulumi.export("analyzers-bucket", analyzers_bucket.bucket)
     model_plugins_bucket = Bucket("model-plugins-bucket", sse=False)
-    pulumi.export("model-plugins-bucket", model_plugins_bucket.bucket)
+    plugin_registry_bucket = Bucket("plugin-registry-bucket", sse=True)
```

Comment:
Are we not exporting this bucket?

Reply:
Upon searching, these exported buckets are never used; somebody was just cargo-culting it. 

If we don't use the export, we shouldn't export it.